utils/sql-db-seeder/src/utilisation-report/utilisation-report.helper.ts

```diff
@@ -99,3 +101,34 @@ export const createUploadedReport = (
 
   return utilisationReport;
 };
+
+export const createFeeRecordsForReport = (): {
+  feeRecordWithMatchingCurrencies: FeeRecordEntity;
+  feeRecordWithDifferingCurrencies: FeeRecordEntity;
+} => {
+  const feeRecordWithMatchingCurrencies = createFeeRecord({
+    facilityId: '12345678',
+    exporter: 'Test exporter',
+    baseCurrency: 'GBP',
+    facilityUtilisation: 1000000,
+    totalFeesAccruedForThePeriod: 100,
+    feesPaidToUkefForThePeriod: 50,
+  });
+
+  const feeRecordWithDifferingCurrencies = createFeeRecord({
```

Comment:
is it worth calling this feeRecordWithDifferingPaymentCurrencies ? Or adding in a totalFeesAccruedForThePeriodCurrency - leaning towards the former for now just because we aren't doing much with differing accruals yet